### DISCUSSION TITLE
Idea for Deadline-feature Word count

### DISCUSSION THREAD

**Discussion by Reproog** [vkbo/novelWriter#1457] [Ideas]
First of all, I really ***LOVE*** novelWriter. I am currently writing my first novel in it.

Is there a possibility to add a "Deadline Word Count Feature" where you can say for example, in so and so many days/weeks/months I want to have reached this number of words (for example 50,000). novelWriter reminds you of this and shows the remaining number of words.

Prominent examples of such a feature would be [Dabble](https://www.dabblewriter.com/), for example.

**Comment by vkbo** [comment]
> First of all, I really _**LOVE**_ novelWriter. I am currently writing my first novel in it.

Thank you!

As, for the word count goals features. There have been a number of different requests for such features. The problem with them is that people have very different ideas of how they want to define their goals and targets. Some people also get very stressed by them. Even the numbers ticking at the end of the status bar. So it's difficult to make something for everyone.

The current GUI gives you the total numbers, and how many words you add in each writing session. My general suggestion is to use the Writing Statistics tool to track your work in the way that suits you.

I've been thinking about adding more features to the Project Details panel and make it a full fledged analysis tool of your project. For now, I have a pretty long list of other features to add, but I'll keep these ideas in mind.

If you have a very concrete idea for a feature, feel free to open a feature issue. It is better if the feature is thought out in a way that can be implemented.

**Reply by yangfancd** [reply]
How about CJK character counting?

**Reply by vkbo** [reply]
There has been a number of requests for different counting methods, so I'm still thinking of a way to implement them all and let the user choose which one is displayed as the primary metric (currently it is hard coded to word count). Please create a feature request for this if it is something you'd like added, and can help with implementing, or at least guide. Doing a quick search revealed this addon for VSCode that may have a useful implementation to copy: https://marketplace.visualstudio.com/items?itemName=holmescn.vscode-wordcount-cjk
